plugins/Diagnostics/Diagnostic/MemoryLimitCheck.php

```diff
@@ -41,7 +41,10 @@ public function execute()
         $memoryLimit = SettingsServer::getMemoryLimitValue();
         $comment = $memoryLimit . 'M';
 
-        if ($memoryLimit >= $this->minimumMemoryLimit) {
+        if(false === $memoryLimit) {
+            $status = DiagnosticResult::STATUS_OK;
+            $comment = 'No memory limit';
```

Comment:
Hi,

Could you maybe expand a bit on that comment and make it translatable (like the other ones)?

Thanks for you contribution.

Reply:
When memory_limit = -1 there is a warning ( screenshot ) 

Here we can see $memoryLimit is set to false when no limit

https://github.com/matomo-org/matomo/blob/4.x-dev/core/SettingsServer.php#L201

I am a newbie on matomo i check the translation guide line

![Capture d’écran de 2021-02-12 10-49-14](https://user-images.githubusercontent.com/5064590/107755020-91f4fe80-6d22-11eb-8265-44eea0a5df9c.png)

Reply:
Sorry, I wasn't precise enough. I just meant make the comment a full sentence, just so noone is confused if this is a suggestion or a statement.